sdk/keyvault/keyvault-common/src/tokenCycler.ts

```diff
@@ -0,0 +1,232 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Could you do me a huge favor and just call out any code here that is _not_ a copy of the core tokenCycler? It'll be easier to review the diffs between this and core that way (and if there's no diff, and just copied verbatim, maybe a comment could explain that)

Reply:
It's a straight copy, I will add a comment to that effect